Ds3/Runtime/Network.cs

```diff
@@ -67,18 +69,22 @@ public IWebResponse Invoke(Ds3Request request)
             using (var content = request.GetContentStream())
             {
                 do
-                {
+                {
+                    if (sdkNetworkSwitch.TraceInfo) { Trace.WriteLine("Request: " + request.GetType().ToString()); }

```

Comment:
Ok, looks like there is the status code already.  So maybe just add how long the request took.